### PR TITLE
fix issue where multiScreenGolden wasn't respecting finder value

### PR DESCRIPTION
**Pull request by CaleyD**
The multiScreenGolden method was ignoring the finder parameter - this fixes that.

### CODE REVIEW

File: packages/golden_toolkit/test/golden_assertion_test.dart [hunk 1]
```diff
@@ -13,9 +13,9 @@ import 'package:golden_toolkit/golden_toolkit.dart';
 
 void main() {
   group('testGoldens validation', () {
-    testWidgets('screenMatchesGolden should require testGoldens',
+    testGoldens('screenMatchesGolden should require testGoldens',
```
Comment (coreysprague) [inline packages/golden_toolkit/test/golden_assertion_test.dart]: was this line intentional?

File: packages/golden_toolkit/test/golden_assertion_test.dart [hunk 2]
```diff
@@ -28,5 +28,34 @@ void main() {
       await expectLater(() => screenMatchesGolden(tester, 'anything.png'),
           throwsAssertionError);
     });
+
+    testGoldens(
+        'Goldens for multiple sized devices should respect specified finder',
+        (tester) async {
+      // This is an example of how a larger widget tree can be pumpded, but goldens can be created capturing just a certain child widget
```
Comment (coreysprague) [inline packages/golden_toolkit/test/golden_assertion_test.dart]:
```suggestion
      // This is an example of how a larger widget tree can be pumped, but goldens can be created capturing just a certain child widget
```

File: packages/golden_toolkit/test/golden_assertion_test.dart (hunk 1 above)
Comment (CaleyD) [inline packages/golden_toolkit/test/golden_assertion_test.dart]:
```suggestion
    testWidgets('screenMatchesGolden should require testGoldens',
```

This was unintentionally committed - here to demonstrate a faulty test (fixed with pumpWidgetBuilder below

**Comment by MattKoppelman** [comment]
@CaleyD @coreysprague Sorry for the review of this PR, I mistook the repository

File: packages/golden_toolkit/test/golden_assertion_test.dart (hunk 1 above)
Comment (CaleyD) [reply packages/golden_toolkit/test/golden_assertion_test.dart]: nope - fixed

**Pull request merged**